department-of-veterans-affairs/caseflow
# Upload backend functionality

Connects #5819 

### Description
This PR adds the basic functionality for creating schedulePeriods from the frontend. Future PRs will address: 
1. Better validation
1. Saving the uploaded file to S3

### Testing Plan
1. Go to hearings/schedule/build/upload
1. Select a file type, enter a start date and an end date, and upload a file
1. Click continue
1. Confirm you're redirected to a new (blank for now) page
1. Confirm the schedulePeriod shows up on /hearings/schedule/build
